docs/develop/make-office-web-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -0,0 +1,85 @@
+
+Specify the `ProgID` of the equivalent COM add-in. Office will then use the COM add-in UI instead of your web add-in UI when the COM add-in is installed.
+
+The following example shows how to specify both a COM add-in and an XLL as equivalent. Often you will specify both so for completeness this example shows both in context. They are identified by their `ProgID` and `FileName` respectively. For more information on XLL compatibility, see [Make your custom functions compatible with XLL user-defined functions](../excel/make-custom-functions-compatible-with-xll-udf.md).
+
+```xml
+<VersionOverrides>
+...
+<EquivalentAddins>  
+<EquivalentAddin>  
+       <ProgID>ContosoCOMAddin</ProgID>  
+       <Type>COM</Type>  
+  </EquivalentAddin>  
+  
+  <EquivalentAddin>  
+       <FileName>contosofunctions.xll</FileName>  
+       <Type>XLL</Type>  
+  </EquivalentAddin>  
+<EquivalentAddins>
+...
+</VersionOverrides>
+```
+
+## Equivalent behavior for users
+
+When an equivalent COM add-in is specified in the web add-in manifest, Office suppresses your web add-in UI on Windows when the equivalent COM add-in is installed. This does not affect your web add-in UI on other platforms like online or macOS. Office only hides the ribbon buttons and does not prevent installation. Therefore your web add-in will still appear in the following UI locations:
+
+- Under **My Add-ins** because it is technically installed.
+- As an entry in the ribbon manager.
+
+The following scenarios describe what happens depending on how the user acquires the web add-in.
+
+### Office Store acquisition of web add-in
+
+If a user downloads the web add-in from the Office store, and the equivalent COM add-in is already installed, then Office will:
+
+1. Install the web add-in.
+2. Hide the web add-in UI in the ribbon.
+3. Display a call-out for the user that points out the COM add-in ribbon button.
+
+### Centralized deployment of web add-in
+
+If an admin deploys the web add-in to their tenant using centralized deployment, and the equivalent COM add-in is already installed, the user needs to restart Office before they will see any changes. After they restart Office, Office will:
+
+1. Install the web add-in.
+2. Hide the web add-in UI in the ribbon.
+3. Display a call-out for the user that points out the COM add-in ribbon button.
+
+### Document shared with embedded web add-in
+
+If a user has the COM add-in installed, and then gets a shared document with the embedded web add-in, then when they open the document, Office will:
+
+1. The user will see a trust prompt for the web add-in.
+2. If trusted, the web add-in will install.
+3. Office hides the web add-in UI in the ribbon.
+
+### User uninstalls the COM add-in
+
+If a user uninstalls the COM add-in, then Office restores the web add-in UI on Windows for the equivalent installed web add-in.
+
+### Web add-in updates are turned off
```

Comment:
Could these last two H3s be merged under a single H2 banner of "Other COM add-in behavior" or something?

Reply:
Agree with Alex!